### PR TITLE
ebs br: new snapshot tagging

### LINKED ISSUE #50559: ebs br: add tag TiDBCluster-BR to snapshots
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Run EBS snapshot backup

### 2. What did you expect to see? (Required)
Snapshot generated has tag with key `TiDBCluster-BR`

### 3. What did you see instead (Required)
The tag is missing

### PR DESCRIPTION
**Pull request by BornChanger**
### What problem does this PR solve?

Issue Number: close #50559 

Problem Summary:

### What changed and how does it work?

1. Add the new tag to snapshots
2. Check fsr enablement periodically instead of sleep for a long time. 

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
image

- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/aws/ebs.go [hunk 1]
```diff
@@ -342,25 +347,7 @@ func (e *EC2Session) waitDataFSREnabled(snapShotIDs []*string, targetAZ string)
...
-
-	// We have to sleep for at least fillElapsedTime minutes in order to make credits are filled to 1.0
-	// Let's heartbeat every 5 minutes
-	for time.Since(start) <= time.Duration(fillElapsedTime)*time.Minute {
-		log.Info("FSR enablement is ongoing, going to sleep for 5 minutes...")
-		time.Sleep(5 * time.Minute)
-	}
-
-	// Wait that all snapshot has enough fsr credit balance, it's very likely true since we have wait for long enough
+	// Wait that all snapshot has enough fsr credit balance
```
Comment (BornChanger) [inline br/pkg/aws/ebs.go:350]: Previously, we will wait based on the estimated fsr enablement time. But if the restore failed, and restarted from scratch, we don't need to wait for such a long time.

**Review by 3pointer — APPROVED** [review]

**Review by YuJuncen — APPROVED** [review]

File: br/pkg/aws/ebs.go (hunk 1 above)
Comment (nkg-) [reply br/pkg/aws/ebs.go:350]: This change doesn't seem related to this PR at all. In general, I encourage you to make separate PRs for logically unrelated PRs.

File: br/pkg/aws/ebs.go [hunk 2]
```diff
@@ -73,6 +73,10 @@ func (e *EC2Session) CreateSnapshots(backupInfo *config.EBSBasedBRMeta) (map[str
 		}
 	}
 
+	tags := []*ec2.Tag{
+		ec2Tag("TiDBCluster-BR", "new"),
```
Comment (nkg-) [inline br/pkg/aws/ebs.go:77]: Looks good.

**Pull request merged**